package.json

```diff
@@ -15,10 +15,11 @@
   },
   "author": "Mudano Ltd. (http://mudano.com)",
   "license": "Apache-2.0",
+  "homepage": "https://mudano.github.io/ui-react/",
   "bugs": {
-    "url": "https://github.com/Mudano/ui-react/issues"
+    "url": "https://mdshowto.atlassian.net/projects/UR/issues",
+    "email": "techops@mudano.com"
```

Comment:
Shouldn't this be the same email address as in the README (`ui-react@mudano.com`)?

Reply:
well spotted! thank you